neutrinordp/xrdp-neutrinordp.c

```diff
@@ -2131,6 +2254,11 @@ mod_init(void)
     freerdp_get_version(&(mod->vmaj), &(mod->vmin), &(mod->vrev));
     LOG(LOG_LEVEL_INFO, "  FreeRDP version major %d minor %d revision %d",
         mod->vmaj, mod->vmin, mod->vrev);
+
+    /* Initialise performance settings masks */
+    mod->perf_settings_override_mask = 0;
+    mod->perf_settings_values_mask = 0;
```

Comment:
I made the initialization of the variables to be done within the mod_init function.
Please let me know if there is another appropriate place.

Reply:
I think you can just remove these.

The structure is allocated with this line:-

```
    mod = (struct mod *)g_malloc(sizeof(struct mod), 1);
```

This make sure the structure is cleared out first. C99 guarantees that an integer type with all bits set to zero will take the value zero, and this assumption is made all over the place in xrdp.

Reply:
As you pointed out.
This part worked correctly after deleting it.
Delete it.